presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSchemaUrl.java

```diff
@@ -103,6 +104,28 @@ public void testHiveCreatedTable(String schemaLocation)
         onHive().executeQuery("DROP TABLE test_avro_schema_url_hive");
     }
 
+    @Test(groups = STORAGE_FORMATS)
+    public void testSymlinkTable()
+            throws IOException
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_avro_schema_url_symlink");
+
+        onHive().executeQuery(format("" +
+                "CREATE TABLE test_avro_schema_url_symlink " +
+                "ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.avro.AvroSerDe' " +
+                "WITH SERDEPROPERTIES ('avro.schema.url'='/user/hive/warehouse/TestAvroSchemaUrl/schemas/original_schema.avsc')" +
```

Comment:
I'd assume it's not important that this table uses avro schema url, is it?
(You can have Avro table with schema url/literal).

Also, this test case doesn't really belong to this class (whihc is focused on schema url handling).

I'd add it in a new class, similar to `TestCsv` class

Reply:
Moved to a new test class